web/src/main/java/co/mobileaction/example/web/service/IUserResultHandlerService.java

```diff
@@ -0,0 +1,12 @@
+package co.mobileaction.example.web.service;
+
+import co.mobileaction.example.common.dto.UserDto;
+
+/**
+ * @author Doga Elif Konuk
+ * @date 17.07.2024
+ * @time 15:15
+ */
+public interface IUserResultHandlerService {
+    void executeMessage(UserDto userDto);
+}
```

Comment:
formatting is also important for readability, there are some different approach in the formatting but we use new lines for the curly brackets in MobileAction. But it differs in other companies, however it is better to keep common formatting throughout the project. Since this is demo project, it isn't that much important.